llama_cpp/llama.py

```diff
@@ -1788,9 +1788,8 @@ def logit_bias_processor(
                 self.detokenize([token]).decode("utf-8", errors="ignore")
                 for token in all_tokens
             ]
-            all_logprobs = [
-                Llama.logits_to_logprobs(row.tolist()) for row in self._scores
-            ][token_offset:]
+            all_logprobs = Llama.logits_to_logprobs(self._scores)[token_offset:]
```

Comment:
`all_logprobs` is now a 2-D array instead of a list of 1-D arrays. But this change should be ok b/c it's zipped, which will iterate over the first axis